src/k8s-engine/wsl.ts

```diff
@@ -1093,8 +1093,21 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
         await this.progressTracker.action(
           this.lastCommandComment,
           100,
-          this.k3sHelper.updateKubeconfig(
-            async() => await this.captureCommand(await this.getWSLHelperPath(), 'k3s', 'kubeconfig')));
+          async() => {
+            // Wait for the file to exist first, for slow machines.
+            const command = 'if test -r /etc/rancher/k3s/k3s.yaml; then echo yes; else echo no; fi';
```

Comment:
`test -r /etc/rancher/k3s/k3s.yaml && echo yes || echo no` works if you want less code

Reply:
I generally dislike `foo && bar || baz`, mainly because it's easy to make a mistake with that form (because the `||` is evaluated even if `&&` was true); it wouldn't have hurt here, though.